app/components/new-project-modal/template.hbs

```diff
@@ -24,13 +24,13 @@
                 {{#if institutions.length}}
                     <div class="form-group m-v-sm NewProject__institution-selector">
                         <label class="f-w-lg text-bigger">{{t 'new_project.affiliation'}}</label>
-                        <a role="button" onclick={{action removeAll}}>{{t 'new_project.remove_all'}}</a>
-                        <a role="button" onclick={{action selectAll}}>{{t 'new_project.select_all'}}</a>
+                        <a role="button" onclick={{action removeAll}} {{action 'click' 'button' 'Dashboard - New Project - remove_all' target=analytics}}>{{t 'new_project.remove_all'}}</a>
+                        <a role="button" onclick={{action selectAll}} {{action 'click' 'button' 'Dashboard - New Project - select_all' target=analytics}}>{{t 'new_project.select_all'}}</a>
```

Comment:
Not important to change now, but I think we should avoid using `onclick={{action}}`, and prefer just the bare `{{action}}`, like you've added. It's confusing to have both, and I don't think it makes a difference (unless there's some subtle interaction with `preventDefault`)

Reply:
We should definitely pick one style and use in consistently. Probably bare actions like @aaxelb said. `onclick={{action}}` sets up a DOM event on the element, while `{{action}}` uses Ember events (which are an abstraction and use a DOM event on the root element). For more info, I highly recommend taking a [Deep Dive on Ember Events](https://medium.com/square-corner-blog/deep-dive-on-ember-events-cf684fd3b808). (Marie also did an excellent talk on this at EmberConf, which should be available soon)

Reply:
So we should make a ticket for this?

Reply:
BTW, Marie recommends using Ember actions over DOM events whenever possible.

Reply:
Ticketed. https://openscience.atlassian.net/browse/EMB-187

Reply:
@brianjgeiger bare actions on `<a>` override the href. Links on staging are currently broken (but do fire the track event). Need to add `onclick=` to those.

Reply:
@binoculars Which links are broken? I just hit a bunch on the dashboard page and they are linking fine, including the 'Select all' and 'Remove all' that this comment is attached to.

Reply:
@binoculars After looking around a bit, it seems like most of the buttons and links work except on the Quickfiles button bar (though the sharing button brings up a dialog). This seems to be more likely your changes than mine. If you're in there anyways, could you capitalize the H in "help"?